pyqtgraph/opengl/GLViewWidget.py

```diff
@@ -381,13 +381,6 @@ def pan(self, dx, dy, dz, relative='global'):
         Prior to version 0.11, *relative* was expected to be either True (x-aligned) or
         False (global). These values are deprecated but still recognized.
         """
-        # for backward compatibility:
-        if isinstance(relative, bool):
-            warnings.warn(
-                "'relative' as a boolean is deprecated, and will not be recognized in 0.13. "
-                "Acceptable values are 'global', 'view', or 'view-upright'",
-                DeprecationWarning, stacklevel=2
-            )    
         relative = {True: "view-upright", False: "global"}.get(relative, relative)
```

Comment:
This line that converts boolean to its equivalent string argument should be removed too

Reply:
nice catch